packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -37,63 +43,48 @@ contract SuperchainWETH is WETH98, ISuperchainWETH, ISemver {
         super.withdraw(wad);
     }
 
-    /// @inheritdoc ISuperchainWETH
-    function sendERC20(address dst, uint256 wad, uint256 chainId) public {
-        // Burn from user's balance.
-        _burn(msg.sender, wad);
-
-        // Burn to ETHLiquidity contract.
-        if (!IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
-            IETHLiquidity(Predeploys.ETH_LIQUIDITY).burn{ value: wad }();
-        }
-
-        // Send message to other chain.
-        IL2ToL2CrossDomainMessenger(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER).sendMessage({
-            _destination: chainId,
-            _target: address(this),
-            _message: abi.encodeCall(this.relayERC20, (msg.sender, dst, wad))
-        });
+    /// @notice Mints WETH to an address.
+    /// @param _guy The address to mint WETH to.
+    /// @param _wad The amount of WETH to mint.
+    function _mint(address _guy, uint256 _wad) internal {
```

Comment:
We should add tests that calls to `SuperchainWETH._mint(address,uint256)` and `SuperchainWETH.mint(address,uint256)` revert (and same for the `_burn` method), to protect against accidentally changing the visibility here, since this must stay internal